postgres/spec/feature_matrix.yaml

```diff
@@ -207,6 +207,11 @@ categories:
         description: |
           Operators may chose by default to generate self-signed SSL certificates.
           They may also offer the option to specify the CA and certificates that users want Postgres clusters to use.
+      - id: mtlsrep
```

Comment:
While not crucially important, so far the convention of having four character `id`s for categories and five character `id`s for the features was used. There's no strong reason to keep it like this --simply there isn't-- but if you believe the IDs like this one and others as part of this commit can be fitted under this convention, it would be better.